src/sync/index.js

```diff
@@ -369,7 +134,10 @@ async function sync(db){
       });
     });
   },
-  async function(){
+  async function() {
+    // Insert Oracle names and options
+    await updateOracleInfo(db, oraclesNeedInfoUpdate);
```

Comment:
is this because Oracle doesnt hv `name` & `options` anymore?

Reply:
correct. removed this and added the name and options when oracles are created.